### PR TITLE
[SYCL] Release notes for July SYCL implementation update.

### PR DESCRIPTION
**Pull request by bader**
Summary of changes between d404d1c6767524c21b9c5d05f11b89510abc0ab9 and 64c0262c0f0b9e1b7b2e2dcef57542a3fe3bdb97.

### CODE REVIEW

File: sycl/ReleaseNotes.md [hunk 1]
```diff
@@ -1,3 +1,80 @@
...
+ - Add new clang built-in function `__unique_stable_name`. SYCL compiler may
+   use this built-in function to auto-generate SYCL kernel name for lambdas.
+
+## Improvements
+ - SYCL integration header is excluded from the dependency list.
+ - Raw pointers capturing added the SYCL device front-end compiler. This
+   capability is required for Unified Shared Memory feature implementation.
+ - SYCL device compiler enabled support for OpenCL types like event, sampler,
+   images to simplify compilation of the SYCL code to SPIR-V format.
+   `CXXReflower` pass used to make "SPRI-V friendly LLVM IR" has been removed.
```
Comment (keryell) [inline sycl/ReleaseNotes.md]: SPIR

File: sycl/ReleaseNotes.md [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
+   address space inference is supposed to be done by LLVM pass.  Old compiler
+   behavior can be recovered by enabling `DISABLE_INFER_AS` environment
+   variable.
+ - Add basic implementation of hierarchical parallelism API.
+ - Add new clang built-in function `__unique_stable_name`. SYCL compiler may
+   use this built-in function to auto-generate SYCL kernel name for lambdas.
+
+## Improvements
+ - SYCL integration header is excluded from the dependency list.
+ - Raw pointers capturing added the SYCL device front-end compiler. This
```
Comment (Ruyk) [inline sycl/ReleaseNotes.md]: added  the-> added to the

File: sycl/ReleaseNotes.md (hunk 1 above)
Comment (bader) [reply sycl/ReleaseNotes.md]: Fixed. Thanks!

File: sycl/ReleaseNotes.md (hunk 2 above)
Comment (bader) [reply sycl/ReleaseNotes.md]: Fixed. Thanks!

File: sycl/ReleaseNotes.md [hunk 3]
```diff
@@ -1,3 +1,80 @@
...
+ - Fixed argument size calculation for zero-dimensional accessor.
+ - Removed incorrect source correlation from kernel instructions leading to
+   incorrect profiling and debug information.
+ - A number of issues were fixed breaking build of the compiler on Windows
+ - Fixed unaligned access in load and store methods of the vector class.
+ - `global_mem_cache_type` values were aligned with the latest revision of the
+   SYCL specification.
+ - Stubs for C++ standard headers were removed. This should fix compilation of
+   <iostream> and <algorithm> with SYCL device compiler.
+ - Unscoped emuns were removed from global namespace to avoid conflicts with
```
Comment (sergey-semenov) [inline sycl/ReleaseNotes.md]:
```suggestion
 - Unscoped enums were removed from global namespace to avoid conflicts with
```

**Pull request merged**